<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Inspector</class>
 <widget class="QWidget" name="Inspector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>287</width>
    <height>519</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Inspector</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="globalFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="filters">
        <property name="spacing">
         <number>0</number>
        </property>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="linkedTags">
        <property name="text">
         <string>Show relation betweend tags</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="linkedRenders">
        <property name="text">
         <string>Show links between tags and recording</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="legend">
        <property name="text">
         <string>Legend</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>HORIZONTAL ZOOM</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="hZoom">
          <property name="decimals">
           <number>1</number>
          </property>
          <property name="minimum">
           <double>0.100000000000000</double>
          </property>
          <property name="maximum">
           <double>100.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>VERTICAL ZOOM</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="vZoom">
          <property name="decimals">
           <number>1</number>
          </property>
          <property name="minimum">
           <double>0.100000000000000</double>
          </property>
          <property name="maximum">
           <double>100.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>319</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>action()</slot>
 </slots>
</ui>
